tests/ClientTest.php

```diff
@@ -68,13 +70,16 @@ public function testConstructClient()
         $client = new MockClient('', $handler);
         $client->runRawQuery('query_string',  false, ['name' => 'val']);
 
+        $client = new MockClient('', $handler, [], ['headers' => [ 'Authorization' => 'Basic xyz', 'User-Agent' => 'test' ]]);
+        $client->runRawQuery('query_string');
+
         /** @var Request $firstRequest */
         $firstRequest = $container[0]['request'];
         $this->assertEquals('{"query":"query_string","variables":{}}', $firstRequest->getBody()->getContents());
 
         /** @var Request $thirdRequest */
         $thirdRequest = $container[1]['request'];
-        $this->assertNotNull($thirdRequest->getHeader('Authorization'));
+        $this->assertNotEmpty($thirdRequest->getHeader('Authorization'));
```

Comment:
@actionm Please assert that the value of the request headers is the one provided in the httpOptions and not the authorizationHeaders after you make the change in the previous comment.